### PR TITLE
test: add some data-cy props

### PR DESCRIPTION
**Pull request by nborde-CSM**

### CODE REVIEW

File: src/misc/ErrorsPanel/ErrorsPanel.js [hunk 1]
```diff
@@ -25,12 +25,12 @@ export const ErrorsPanel = (props) => {
...
   return (
-    <Paper className={classes.errorsContainer}>
-      <Typography className={classes.errorsHeader}>
+    <Paper className={classes.errorsContainer} data-cy="errors-panel">
+      <Typography className={classes.errorsHeader} data-cy="errors-header">
         {labels.mainError} {errorsCountLabel}
       </Typography>
       {errors.map((error, index) => (
-        <Accordion key={'error' + index}>
+        <Accordion key={'error' + index} data-cy="error-accordion" data-cy-index={'error-accordion-' + index}>
```
Comment (csm-thu) [inline src/misc/ErrorsPanel/ErrorsPanel.js:33]: Do we really need both `data-cy` and `data-cy-index` properties ?
If we only set `data-cy={'error-accordion-' + index}`, it seems we can still select all elements whose data-cy starts with "error-accordion" with a selector like `cy.get('[data-cy^=error-accordion]')` (I have not tested it yet though)

File: src/misc/ErrorsPanel/ErrorsPanel.js [hunk 2]
```diff
@@ -43,10 +43,14 @@ export const ErrorsPanel = (props) => {
               </Typography>
             </Box>
             <Box flexGrow={1}>
-              <Typography className={classes.errorSummary}>{error.summary}</Typography>
+              <Typography dataclassName={classes.errorSummary} data-cy={'error-summary'}>
```
Comment (csm-thu) [inline src/misc/ErrorsPanel/ErrorsPanel.js:46]: You changed `className` to `dataclassName`, this will break the style of these elements :sweat_smile:

**Review by csm-thu — CHANGES_REQUESTED** [review]

File: src/misc/ErrorsPanel/ErrorsPanel.js (hunk 2 above)
Comment (nborde-CSM) [reply src/misc/ErrorsPanel/ErrorsPanel.js:46]: It's a mess...

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**